Tools/RemoteConsole/remote_console.py

```diff
@@ -5,54 +5,91 @@
 SPDX-License-Identifier: Apache-2.0 OR MIT
 """
 
-import sys
-import signal
 import argparse
 import logging
 import ly_remote_console.ly_remote_console.remote_console_commands as remote_console_commands
+import os
+import signal
+import sys
+import threading
+import time
 
 client_message_logger = logging.getLogger('remote_console.client_message')
 client_message_logger.setLevel(logging.INFO)
-message_handler = logging.FileHandler('remote_console.log', mode='w+', encoding='utf-8')  # use mode='a' to append
-message_formatter = logging.Formatter(fmt='%(asctime)s %(message)s', datefmt='%m/%d/%Y %I:%M:%S %p')
-message_handler.setFormatter(message_formatter)
-client_message_logger.addHandler(message_handler)
 
 diagnostic_logger = logging.getLogger('remote_console.diagnostic')
-diagnostic_handler = logging.StreamHandler(stream=sys.stdout)
 diagnostic_logger.setLevel(logging.DEBUG)
+diagnostic_handler = logging.StreamHandler(stream=sys.stdout)
 diagnostic_logger.addHandler(diagnostic_handler)
 
-remoteConsole = None
+remote_console = None
+messages_to_console = False
+
+
+def inputCommand():
+    global remote_console
+    global diagnostic_logger
+
+    while remote_console.connected:
+        command = input()
+        if remote_console.connected:
+            remote_console.send_command(command)
+            diagnostic_logger.info(f'sent command "{command}"')
 
 
 def main():
-    global remoteConsole
+    global remote_console
+    global client_message_logger
+    global messages_to_console
 
     parser = argparse.ArgumentParser(description="Provides common dev functionality", add_help=False)
     parser.add_argument('--addr', default='127.0.0.1', dest='address', help="Address for Remote Console to connect")
     parser.add_argument('--port', default='4600', dest='port', help="Port for Remote Console to connect")
+    parser.add_argument('--console', action='store_true', dest='console', help="All the messages to console instead of remote_console.log file")
```

Comment:
The help message here is a little vague. This sends commands to the remote console when using --console correct? It doesn't just cause messages that would normally go to the remote_console.log file to be displayed in the console, right? Also sends commands yes?

Reply:
--console is to print client's log messages to console. Junho wanted this.
Sending commands is independent from the --console command line key, it works any case.